plugins/poll/test/javascripts/component/poll-results-ranked-choice-test.js

```diff
@@ -31,6 +31,8 @@ const RANKED_CHOICE_OUTCOME = {
   ],
 };
 
+const EMPTY_RANKED_CHOICE_OUTCOME = {};
```

Comment:
Is the constant necessary here?

Reply:
Not necessary, just trying to make the test more readable and obvious.

Can remove.

Reply:
Removed https://github.com/discourse/discourse/pull/28542/commits/b820fc17c7f94a7fc8e17f77089d2addf6d5c4b7